### ISSUE TITLE
Isn't grunt a dev dependency, not a prod dependency

### ISSUE THREAD

**Issue by ndushay** [LD4P/sinopia_profile_editor#17]
See

https://github.com/ProjectMirador/research-and-demos/blob/master/minimal_redux_poc/package.json

note that babel and webpack are dev dependencies ...

**Comment by atz** [comment]
`grunt` is a **build** dependency, afaict.

**Comment by ndushay** [comment]
yes, it's a build dependency, but not a run dependency, which is what I meant by "prod" -- once it's built, it's not needed
